batavia/types/StrUtils.js

```diff
@@ -1233,11 +1230,15 @@ function _new_subsitute(str, args, kwargs) {
         // error for converting floats with improper presentation types
         // TODO: need to check for decimal once it is implemented
         if (types.isinstance(this.arg, [types.Float]) && /[bcdoxX]/.test(type)) {
-            throw new types.ValueError.$pyclass(`Unknown format code '${type}' for object of type '${type_name(this.arg)}'`)
+            throw new exceptions.ValueError.$pyclass(`Unknown format code '${type}' for object of type 'float'`)
         }
 
-        if (this.type === 'c' && this.sign) {
-            throw new exceptions.ValueError.$pyclass("Sign not allowed with integer format specifier 'c'")
+        if (this.type === 'c') {
+            if (this.sign) throw new exceptions.ValueError.$pyclass("Sign not allowed with integer format specifier 'c'")
```

Comment:
Can you change these ifs to use braces for the blocks? It's not in the .eslintrc, but it is an oversight.